examples/stable/test/end_to_end/candid_rpc/candid_backtraces/test/test.ts

```diff
@@ -0,0 +1,139 @@
+import { test } from '@jest/globals';
+import { getCanisterId } from 'azle/_internal/dfx';
+import { expect,runTests } from 'azle/_internal/test';
+import { execSync } from 'child_process';
+
+import { createActor } from './dfx_generated/candid_backtraces';
+
+const canisterName = 'candid_backtraces';
+const testMessage = 'Test trap message for canister backtraces';
+
+function deployCanisterWithEnv(envValue?: string): void {
```

Comment:
Remember that these types of utility/helper functions should be at the bottom of the file.